.github/workflows/sdk-generation.yaml

```diff
@@ -180,7 +181,7 @@ jobs:
       - name: Tune GitHub-hosted runner network
         uses: smorimoto/tune-github-hosted-runner-network@v1
       - id: validate
-        uses: speakeasy-api/sdk-generation-action@v14
+        uses: speakeasy-api/sdk-generation-action@a1bec757ff7ddd2b0a83ffdf7c12bf7683cd5928
```

Comment:
this will need to go back to @v14 before merging

Reply:
yup, just added for testing since i thought some of my changes wouldn't be used here if i was referencing `@v14`